i18n/convert.go

```diff
@@ -26,7 +26,7 @@ var javaFormatPlaceholderRegexp = regexp.MustCompile(`{(\d)}`)
 // FromJavaToGoSyntax convert a translation string made for Java to a one suitable for golang (printf-style).
 // The conversion transforms java placeholders like "{0}","{1}","{2}",etc... with the equivalent for golang
 // "%[1]v","%[2]v","%[3]v",etc...
-// The double single-quote "''" is translated into a single single-quote "'".
+// A pair of adjacent single-quote characters "'" (U+0027) is replaced with one only.
```

Comment:
required rewriting for the formatter to not replace "''" with "”" and making checks angry.
See https://tip.golang.org/doc/comment#paragraphs